src/main.rs

```diff
@@ -64,6 +64,9 @@ async fn handle_message(
         }
     }
 
+    // Assets (NFT)
+    db_adapters::assets::non_fungible_token_events::store_nft(&pool, &streamer_message).await;
```

Comment:
Instead of awaiting immediately, create a future and add it to the `join!`s below, so we can do things in parallel.